Generate-TranslationNuGetPackages.ps1

```diff
@@ -0,0 +1,76 @@
+﻿$localizationFolder = "Localization";
+$culturesFolders = $(Get-ChildItem "$localizationFolder");
+
+$packageExtension = ".nuget";
+$packageNamePrefix = "OrchardCore.Translations.";
+$PackageVersionNumber = "1.0.0";
+
+$NuGetSpecExtension = ".nuspec";
+$NuGetSpecTemplate = "NuGetSpecTemplate" + $NuGetSpecExtension;
+
+$PackagesFolder = "NuGet Packages";
+
+echo "Start generating translation NuGet packages ..";
+
+# Create NuGet folder if not exist
+createNuGetFolder;
+
+foreach($cultureFolder in $culturesFolders) {
+    # Generate tranlation NuGet package
+    createNuGetPackage $cultureFolder.Name;
+}
+
+echo "Done!!";
+
+function createNuGetFolder()
+{
+    if(-Not(Test-Path -Path $PackagesFolder))
+    {
+        New-Item -Path $PackagesFolder -ItemType "Directory";
+    }
+    else
+    {
+        # Remove old translation packages
+        Remove-Item -Path $PackagesFolder\* -Recurse -Force;
+    }
+}
+
+function prepareNuGetSpec($culture)
+{
+    $packageName = $packageNamePrefix + $culture;
+    $NuGetSpecFileName = $packageName + $NuGetSpecExtension;
+    $NuGetSpecFilePath = [IO.Path]::Combine($PWD, $localizationFolder, $culture, $NuGetSpecFileName);
+
+    $nugetSpec = [xml](Get-Content -Path $NuGetSpecTemplate);
+    $metadata = $nugetSpec.package.metadata;
+    $metadata.id = $packageName;
+    $metadata.version = $PackageVersionNumber;
+    $metadata.authors = "Hisham Bin Ateya";
```

Comment:
What the authors & owners should be?

Reply:
The Orchard Team

Reply:
Can it be in the template directly? same for the version number and we can just inject the build number in this template?

Reply:
Of course it will be in the template, regarding the version I'm still thinking about better way if there's